### PR TITLE
rm home-grown set helper, use ember-set-helper instead.

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes https://github.com/ilios/ilios/issues/5663

see https://github.com/ilios/frontend/commit/6da92bab25ed756f0e154058d4a2b4a5fd44a568

to check, use this grep-chain in the repo's root:

```bash
egrep -lr '({{|\()set this\.' packages/ | egrep -v '(dist|node_modules)' | egrep '(hbs|tests)'
```

### CODE REVIEW

**Comment by jrjohnson** [comment]
The `ember-set-helper` implementation has a different API than our version:
Us:
`{{set this.foo 'bar'}}`
Them
`{{set this 'foo' 'bar'}}`

Our transform and `ember-simple-set` before us goes through and extracts the `.` (dot) so we can do `this.foo` instead of `this "foo"`. I'd be ok with making this change, but it's a big lift.

**Comment by stopfstedt** [comment]
i'm shelving this for the moment, but plan on getting back into it next week.

**Comment by stopfstedt** [comment]
regex search/replace for the win.

**Review by jrjohnson — CHANGES_REQUESTED** [review]
Excellent. I hate this syntax, but love getting rid of our special ❄️ transform.

File: packages/frontend/.lint-todo [hunk 1]
```diff
@@ -62,3 +62,5 @@ add|ember-template-lint|no-at-ember-render-modifiers|10|6|10|6|d919d2af254f782c0
 add|ember-template-lint|no-at-ember-render-modifiers|11|6|11|6|940005188b476a060b0e5d3f05baea24ba178878|1722902400000|1730682000000|1754006400000|app/components/reports/subject/new/session-type.hbs
 add|ember-template-lint|no-at-ember-render-modifiers|10|6|10|6|d919d2af254f782c01fe2ba15416673e52e91124|1722902400000|1730682000000|1754006400000|app/components/reports/subject/new/term.hbs
 add|ember-template-lint|no-at-ember-render-modifiers|11|6|11|6|940005188b476a060b0e5d3f05baea24ba178878|1722902400000|1730682000000|1754006400000|app/components/reports/subject/new/term.hbs
+remove|ember-template-lint|no-at-ember-render-modifiers|5|4|5|4|ddf86727990ead62165fe28c6dff53552a4d49b7|1722902400000|1730682000000|1754006400000|app/components/user-menu.hbs
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply packages/frontend/.lint-todo]: handled. :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**